src/lib/components/atoms/ApiSettingsButton.svelte

```diff
@@ -0,0 +1,123 @@
+<script lang="ts">
+	import * as settings from '../../../data/api/settings';
+	import { clear } from '../../../data/api/cache';
+	let url = settings.api_base_url();
+	function handler() {
+		clear();
+		settings.set_api_base_url(url);
+		location.reload();
+	}
+	function reset() {
+		url = settings.default_base_url;
+	}
+	import Modal from '$lib/components/atoms/Modal.svelte';
+</script>
+
+<Modal icon="/icons/settings.svg">
+	<div class="settings-menu">
+		<h3>Configure the backend</h3>
+		<div class="url-stuff">
+			<h4>Api URL:</h4>
```

Comment:
```suggestion
			<h4>API URL:</h4>
```
Just to make it consistent with flutter manager :)